src/test/java/com/daml/extensions/testing/ledger/SandboxManagerIT.java

```diff
@@ -66,7 +74,7 @@ public void managerReturnsAutomaticallyAssignedLedgerId() throws Exception {
             false,
             Optional.empty(),
             Optional.empty());
-    String ledgerIdPattern = "[a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}";
+    String ledgerIdPattern = "sandbox"; // todo elaborate
```

Comment:
why?

Reply:
Initialy the ledgerId with this pattern was assigned by  LedgerIdentityServiceGrpc. This code was serves this purpose inside SandboxManager.java - 
   
```
    String ledgerId =
        LedgerIdentityServiceGrpc.newBlockingStub(channel)
            .getLedgerIdentity(
                LedgerIdentityServiceOuterClass.GetLedgerIdentityRequest.newBuilder().build())
            .getLedgerId();
```

But today LedgerIdentityServiceGrpc is depricated. Thats why I substitute it with simple "sanbox" (see new SandboxManager.java) and check it in the test

Reply:
Is there currently no way with the new sandbox to set the ledger ID?

Reply:
https://docs.daml.com/app-dev/grpc/proto-docs.html#ledgeridentityservice

LedgerIdentityService
DEPRECATED: This service is now deprecated and ledger identity string is optional for all Ledger API requests.

Maybe then we should not use the ledgerId anywhere at all.

Reply:
right.. The problem is that the ledgerId property is all over the place in the library. I didn't touch it because a lot of stuff can go broken after it. I'll try to get rid of it when I resolve other problems

Reply:
thank you

Reply:
btw @vshneer if the Ledger Identity Service / API is "just" deprecated, that does not mean it is already gone, right? So we can still use it (keeping it mind that it will go away soon).